src/components/Dropdown.astro

```diff
@@ -0,0 +1,62 @@
+---
+import Icon from "astro-icon";
+import classNames from "classnames";
+
+type Props = {
+  item: string;
+  options: readonly string[];
+  onOptionClick: (option: string) => string | undefined;
+};
+
+const { item, options, onOptionClick } = Astro.props;
+---
+
+<div class="min-w-[6rem]">
+  <div class="relative mt-1">
+    <button
+      type="button"
+      class="peer relative w-full cursor-default rounded-md py-2 pl-3 pr-10 text-left text-white hover:bg-wsg-orange-400 hover:bg-opacity-50 focus:ring-1 focus:ring-wsi-blue-300 sm:text-sm"
+      aria-haspopup="listbox"
+      aria-expanded="true"
+      aria-labelledby="listbox-label"
+    >
+      <span class="block truncate text-base font-medium">{item}</span>
+    </button>
+    <ul
+      class="absolute z-10 mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm opacity-0 pointer-events-none hover:pointer-events-auto peer-focus:pointer-events-auto peer-focus:opacity-100 transition-opacity"
+      tabindex="-1"
+      role="listbox"
+      aria-labelledby="listbox-label"
+      aria-activedescendant="listbox-option-3"
+    >
+      {
+        options.map((option) => {
+          const isSelected = option === item;
+
+          return (
+            <li
+              onclick={isSelected ? undefined : onOptionClick(option)}

```

Comment:
a bit shorter and more readable syntax

```suggestion
              onclick={isSelected && onOptionClick(option)}

```

Reply:
![grafik](https://user-images.githubusercontent.com/45438553/214026604-40e4d017-ac68-408f-a143-4f69d13d2b28.png)

Reply:
Nevermind 😆